collector/src/main/java/com/bitmovin/analytics/data/DeviceInformationProvider.kt

```diff
@@ -4,20 +4,32 @@ import android.app.UiModeManager
 import android.content.Context
 import android.content.res.Configuration
 import android.os.Build
+import android.util.DisplayMetrics
+import android.view.WindowManager
 import com.bitmovin.analytics.utils.Util
+import kotlin.math.roundToInt
 
 open class DeviceInformationProvider(val context: Context, val userAgent: String) {
     var isTV: Boolean = isTVDevice()
 
     fun getDeviceInformation(): DeviceInformation {
+
+        val windowManager: WindowManager = context.getSystemService(Context.WINDOW_SERVICE) as WindowManager
+        val displayMetrics = DisplayMetrics()
+
+        windowManager.defaultDisplay.getMetrics(displayMetrics)
```

Comment:
Can WindowManager be `null`?

Reply:
I believe that `WindowManager` is non nullable type. As long as `context` is not null, everything should be fine. `getSystemService()` method should return null only in cases when some random string is passed as name parameter, and service with that name doesn't exist.

We already use `context.getSystemService()` in `isTVDevice()` method, so I think it should be safe to use it here.

Reply:
probably should get a guard or something here

Reply:
Added null check in https://github.com/bitmovin/bitmovin-analytics-collector-android/pull/118/commits/02c97d8446258f0af46b08139bf4fd3cebc34db6